### PR TITLE
fix selector for explainer body typography placeholder

### PR DESCRIPTION
**Pull request by leezaweb**

### CODE REVIEW

File: assets/_sass/spotlights/components/_explainer.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
       position: fixed;
       top: 0;
       left: 0;
-      z-index: 10;
+      z-index: 9999;
```
Comment (jnschrag) [inline assets/_sass/spotlights/components/_explainer.scss]: Why was this changed?

Comment (leezaweb) [reply assets/_sass/spotlights/components/_explainer.scss]: @jnschrag The exit icon was covered by the header, so I adjusted this z-index, to 9999.
I took that number from the clippy-greg component which is also stuck at 0 on mobile. And I probably got that number from the definition tooltip powered, by Tippy and the Tippy people chose 9999 for that.

**Comment by leezaweb** [comment]
Just remembered... I know the INCLUDES file needs to be updated, but do you want to keep an unpublished page where we can see all available components?
